assets/app/js/controllers/scanDataTable.js

```diff
@@ -340,15 +338,11 @@ class ScanDataTable extends Controller {
                 { $eq: null }
               ]
             };
-          } else if (exactFilterColumnNames.includes(columnName)) {
-            query.where.$and[columnName] = {
-              $eq: columnSearch
-            };
           } else {
             query.where.$and[columnName] = {
-              $like: `%${columnSearch}%`
+              $eq: columnSearch
```

Comment:
Unfortunately, this will break quite a few filter behaviors. There was a reason why we used a list of values above: Some columns will not filter correctly without a `$like`.

Reply:
If I back with previous code, in that case only three columns are filtered with ```$eq``` else filtered the data using ```$like``` in that case it uses sub-string rather then using exact value which we provided as a list on the top of column.

But @majurg, my doubt is if we already provide a value on the top of column then why we need of ```$like```` query? if it breaks quite filters so will you provide me a test case where it break a filter behavior? 

If it breaks the behavior of filter using ```$eq``` then the solution of this issue is to update the list of   
```
const exactFilterColumnNames = ['extension', 'programming_language', 'name'];
```
and append the list to columns and if filter is break on that case then use ```$like``` rather then ```$eq```

Reply:
Some columns, like license_key, are lists of values at this time. It turns out that filtering with `$eq` will not work for lists of values, as it expects to match against the entire list as opposed to a single element. 

This is not great from a schema standpoint, I will admit, but at this time this is how things work.